assembly-syntax/src/ast/instruction/advice.rs

```diff
@@ -15,6 +15,8 @@ pub enum SystemEventNode {
     PushU64Div,
     PushFalconDiv,
     PushSmtPeek,
+    PushArrayLowerBound,
+    PushMapLowerBound,
```

Comment:
I'm not sure about this naming here: I think `PushArrayLowerBound` is fine, but maybe instead of `PushMapLowerBound` it should be `PushKeyValueLowerBound`?